### PR TITLE
Add Changes 

### PR DESCRIPTION
**Pull request by GiorgosM1995**
This is the correct changes

### CODE REVIEW

File: vendors/cicicom/model/s-lg-i3/model.yaml
Comment (tomHallActility) [inline vendors/cicicom/model/s-lg-i3/model.yaml:1]: Hi Cicicom,

Could you remove the "product" field (including name and version) as they are now deprecated.
Also, please make the modelId lowercase only.

Thank you very much.

File: vendors/cicicom/profiles/profile.yaml
Comment (tomHallActility) [inline vendors/cicicom/profiles/profile.yaml:1]: In order to keep your profiles separate in the future, please rename profile.yaml as "cicicom_RFGroup1_1.0.2a_classA.yaml".
Please change the id in the file as "cicicom_RFGroup1_1.0.2a_classA", as well.

Thank you in advance.

**Review by tomHallActility — CHANGES_REQUESTED** [review]
Hello Cicicom,

Thank you for correcting your original pull request.
Here are a couple more things to change. 
Please take your time to review them.

Thank you very much.

Best regards,
Tom

File: vendors/cicicom/model/s-lg-i3/model.yaml
Comment (tomHallActility) [inline vendors/cicicom/model/s-lg-i3/model.yaml:1]: Hi,
I believe there's a small typo line 27: 
icicom:s-lg-i3:2 -> cicicom:s-lg-i3:2

File: vendors/cicicom/profiles/cicicom_RFGroup1_1.0.2a_classA.yaml
Comment (tomHallActility) [inline vendors/cicicom/profiles/cicicom_RFGroup1_1.0.2a_classA.yaml:1]: How we link the profiles to the models is through the id field line 11.
Please change:
lora-generic_RFGroup1_1.0.2a_classA
to
cicicom_RFGroup1_1.0.2a_classA
Thank you very much.

**Review by tomHallActility — CHANGES_REQUESTED** [review]
Hello Giorgos,

I hope you are doing well.
Thank you for correcting your previous pull request.
There are still a couple of things to change.
Thank you for taking a look at them.

Have a great day,
Tom

File: vendors/cicicom/model/s-lg-i3/model.yaml
Comment (tomHallActility) [inline vendors/cicicom/model/s-lg-i3/model.yaml:1]: Hello Giorgos,

There is still one small correction to make, can you change the protocolId:
protocolId: cicicom:s-lg-i3
to
protocolId: cicicom:s-lg-i3:1

Thank you.

**Review by tomHallActility — CHANGES_REQUESTED** [review]

File: vendors/cicicom/model/s-lg-i3/model.yaml
Comment (GiorgosM1995) [reply vendors/cicicom/model/s-lg-i3/model.yaml:1]: Hello Tom,
thanks for answering I made the correction

**Review by tomHallActility — APPROVED** [review]

**Pull request merged**